applications/MeshingApplication/tests/test_MeshingApplication_mpi.py

```diff
@@ -24,6 +25,8 @@ def AssembleTestSuites():
     ### Small MPI tests ########################################################
     smallMPISuite = suites['mpi_small']
     smallMPISuite.addTest(TestMPIMetrics('test_mpi_hessian'))
+    if hasattr(KM.MeshingApplication,  "ParMmgProcess3D"):
```

Comment:
Global comment (not for this PR):
in the LinearSolversApp we also had to use `hasattr` quite often, and ended up doing it better/properly. I think the same solution could be applied here:
https://github.com/KratosMultiphysics/Kratos/blob/475f6d3581c6ea1e9d2faa3bbb50059d55c23404/applications/LinearSolversApplication/custom_python/eigen_solvers_python_application.cpp#L37-L50

Reply:
I like it, I will put it as to do in the roadmap issue (also the serial version could be improved with this)